core/spec/models/spree/order_contents_spec.rb

```diff
@@ -54,6 +56,14 @@
       expect(order.total.to_f).to eq(19.99)
     end
 
+    it "should create stock location associations if provided" do
+      line_item = subject.add(variant, 3, stock_location_quantities: {stock_location.id => 1, stock_location_2.id => 2})
```

Comment:
Line is too long. [120/100]<br>Space inside { missing.<br>Space inside } missing.